src/main/java/io/aiven/kafka/connect/s3/S3SinkConfig.java

```diff
@@ -122,6 +122,7 @@
     private static final String GROUP_FILE = "File";
     private static final String GROUP_FORMAT = "Format";
     private static final String DEFAULT_FILENAME_TEMPLATE = "{{topic}}-{{partition}}-{{start_offset}}";
+    public static final String PROPERTIES_FILENAME = "aiven-kafka-connect-s3-version.properties";
```

Comment:
This constant probably doesn't belong here. `S3SinkConfig` is about connector configuration and the version is not a part of it.